app/assets/stylesheets/components/top-bar.scss

```diff
@@ -4,50 +4,86 @@ $main-tiles: 7;
 .top-bar {
   @include outer-container;
   max-width: 100%;
-  background: image-url('background-pattern.jpg') 0 3px;
   clear: both;
+  position:relative;
+  top:-2px;
 
   div {
-    display: inline;
     margin: 0;
     padding: 0;
   }
 }
 
-.skyline {
-  margin-top: 1px;
-}
 
 .top-bar__logo {
-  float: left;
+border-top:1px solid #e9e3c8;
+text-align:center;
+background:url("skyline2.png") repeat-x;
```

Comment:
In the "real world" of Rails apps, a weird MD5 hash is appended to all image files. So this will actually look more like `skyline2-2308408234802304230847802340823084230894.png` when we deploy. There's a special SASS function you can use called `asset-url` that you can use to get the hash in your filename:

```css
background: asset-url('skyline2.png') repeat-x;
```